src/Ultz.SuperInvoke.Loader/LibraryLoader.cs

```diff
@@ -20,9 +20,28 @@ public abstract class LibraryLoader
         /// </summary>
         /// <param name="name">The name of the library to open.</param>
         /// <returns>The operating system handle for the shared library.</returns>
-        public IntPtr LoadNativeLibrary(string name)
+        public IntPtr LaodNativeLibrary(string name)
```

Comment:
Oh whoops there was a typo, one last thing before merge!

Reply:
Good catch!!
Was trying to slam it out before work started, hence the mistakes.  👍

Reply:
fixed